.github/ISSUE_TEMPLATE/launch.md

```diff
@@ -0,0 +1,154 @@
+---
+name: Launch
+about: Prepare for a site launch with a checklist of common steps.
+title: ''
+labels: 'devops'
```

Comment:
I've been trying to avoid adding labels to create dependencies (even though I don't think it hurts anything), as it's not that hard to apply your own labels, especially when you only create this issue once per site. So I'm inclined to say no, but also 🤷‍♂️ .

Reply:
I get that. `dept: devops 🚧` seems to be a standard we use across several projects (do we import those from somewhere?), so I think it makes about as much sense as `devops`. That said, I don’t feel too strongly about this. I’m fine with leaving this as is.

Reply:
> I get that. dept: devops 🚧 seems to be a standard we use across several projects (do we import those from somewhere?)

The problem as I see it is that labels are not standardized across projects. They sometimes vary wildly but more often just in small ways (emoji or no emoji). There was not previously a way to standardize labels, but in my due diligence while writing this comment, I discovered that [GitHub rolled out some functionality around this last year: [Managing default labels for repositories in your organization](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/managing-default-labels-for-repositories-in-your-organization). This will only work for new repos though, we would have to manually update existing ones.

These are our options as I see it.

1. Include no labels in the template. Let repos specify their own.
1. Include a sane default. If there is not a matching label in the repo, it will just ignore it. In those cases, it is effectively the same as option 1.
1. Include multiple derivatives of each label. If any match they will apply those.
1. [Standardize our labels](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/managing-default-labels-for-repositories-in-your-organization) for new repos and manually update existing ones and then come back and update this template once that is done.

Reply:
@markdorison beat me to it, but I was going to propose option 1. I didn't realize that I had applied the `devops` label.

I really don't care too much though, and option 1 seems like the path of least resistance.

Reply:
That sounds good to me.

Reply:
I like option no. 4 the most, but that’s the path of most resistance and can be handled at a later date. For now, I think option no. 3 is my second favorite.

But again, if we leave it as-is I’m cool with it.

Reply:
I just went with 1 for now. Follow-up PR's are welcome. We can discuss in our upcoming call too.